src/operators/Cylinder/PyCylinderAttributes.C

```diff
@@ -316,8 +316,15 @@ PyCylinderAttributes_setattr(PyObject *self, char *name, PyObject *args)
         Py_DECREF(obj);
```

Comment:
Note that this may be problematic as `NULL` is *never* returned and `obj` is never initialized as `NULL`. But, I think the `Py_DECREF(obj)` is harmless when `obj` has any of the `PyExc_XXXError` values.